src/commands/synthetics/run-tests-command.ts

```diff
@@ -185,12 +202,42 @@ export class RunTestsCommand extends Command {
     let results: Result[]
     let summary: Summary
 
+    const [_, command] = this.path ?? []
+    if (command === 'build-and-test') {
+      if (!this.config.buildCommand) {
+        this.reporter.error('The `buildCommand` option is required for the `build-and-test` command.')
+
+        return 1
+      }
+      if (!this.config.buildPluginPort) {
+        this.reporter.error('The `buildPluginPort` option is required for the `build-and-test` command.')
+
+        return 1
+      }
```

Comment:
You can declare the relationship of both `--buildCommand` and `--buildPluginPort` using Typanion's [`hasKeyRelationship`](https://mael.dev/typanion/docs/predicates/cascading#haskeyrelationship) directly in the command's [`schema`](https://mael.dev/clipanion/docs/validation#validating-commands).

```
static schema = [
  t.hasKeyRelationship('buildCommand', t.KeyRelationship.Requires, ['buildPluginPort'])
]
```

TBD the usefulness of `buildPluginPort` altogether 😄

Reply:
I went ahead and removed the `buildPluginPort` altogether.